source/variables/custom-selectors.css

```diff
@@ -1 +1,8 @@
-@custom-selector :--enter :hover, :active;
\ No newline at end of file
+/* 
+	Allows you to create your own selectors
+	http://cssnext.io/features/#custom-selectors
+	:--enter {
+	  ... styles ...
+	}	
+*/
+@custom-selector :--enter :hover, :focus, :active;
```

Comment:
We either don't want focus on this selector, or to have a separate custom selector for each variation.  Focus and Active are different actions.  :hover, :active is generally going to be mouse/pointer based,  where :focus is going to be keyboard based.  http://stackoverflow.com/questions/1677990/what-is-the-difference-between-focus-and-active